Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -55,12 +55,13 @@ void GeneralWidget::CreateWidgets()
   m_video_layout = new QGridLayout();
 
   m_backend_combo = new ToolTipComboBox();
-  m_aspect_combo =
-      new GraphicsChoice({tr("Auto"), tr("Force 16:9"), tr("Force 4:3"), tr("Stretch to Window")},
-                         Config::GFX_ASPECT_RATIO);
+  m_aspect_combo = new GraphicsChoice(
```

Comment:
If this feature is kept, I think we would want a different name.   Unfortunately, I'm terrible with names, I don't have any suggestions on it.  Regardless, this needs to be added to the description as well.